src/polly/src/epoll.rs

```diff
@@ -160,7 +183,12 @@ impl Epoll {
     /// (measured in milliseconds).
     /// * `events` points to a memory area that will be used for storing the events
     /// returned by `epoll_wait()` call.
-    pub fn wait(self, max_events: usize, timeout: i32, events: &mut [Event]) -> io::Result<usize> {
+    pub fn wait(
+        self,
+        max_events: usize,
```

Comment:
Yap, the scenario is somewhat similar to what we have in Firecracker. You preallocate a buffer of size T that you always use when calling `wait` so that you don't have to do allocations every time you exit the loop. If you don't want to wait for `T` events, but `X` events (where `X` < `T`) there is no other way for you to specify the size.

P.S. This code was already merged in a previous PR. The changes in this PR are style only as I renamed Event to EpollEvent hence cargo fmt complained about it.